python/sklearnserver/sklearnserver/model.py

```diff
@@ -28,8 +29,15 @@ def __init__(self, name: str, model_dir: str):
         self.ready = False
 
     def load(self):
-        model_file = os.path.join(kfserving.Storage.download(self.model_dir), JOBLIB_FILE) #pylint:disable=c-extension-no-member
-        self._joblib = joblib.load(model_file) #pylint:disable=attribute-defined-outside-init
+        model_path = kfserving.Storage.download(self.model_dir)
+        joblib_path = os.path.join(model_path, JOBLIB_FILE)
```

Comment:
consider the alternate form:

paths = [
  os.path.join(model_path, JOBLIB_FILE),
  os.path.join(model_path, PICKLE_FILE)
]
model_file = next(path for path in paths if os.path.exists(path))
self._model = joblib.load(model_file)

Reply:
I updated the loading method with this form, thank you for the suggestion.

Reply:
Thanks @Tomcli 
/lgtm